formats/json/commonTest/src/kotlinx/serialization/json/BasicTypesSerializationTest.kt

```diff
@@ -36,8 +36,8 @@ class BasicTypesSerializationTest : JsonTestBase() {
         testPrimitive("string", "\"string\"", useStreaming)
     }
 
-    private inline fun <reified T : Any> testPrimitive(primitive: T, expectedJson: String, useStreaming: Boolean) {
-        val json = default.encodeToString(primitive, false)
+    private inline fun <reified T : Any> testPrimitive(primitive: T, expectedJson: String, useStreaming: JsonTestingMode) {
+        val json = default.encodeToString(primitive, JsonTestingMode.TREE)
```

Comment:
It seems to be a typo from the previous version of the code, `useStreaming` should be used.
Also, it's worth renaming it to `jsonTestingMode` here

Reply:
And in our test-specific `decode*` and `encode*` methods as well